### PR TITLE
Adds EligConditions for IHSS.

### PR DESCRIPTION
**Pull request by rajivrajiv**
ihssResult() updated.

### CODE REVIEW

File: src/site/_includes/js/eligibility.js [hunk 1]
```diff
@@ -1782,16 +1782,15 @@ function ihssResult(input) {
...
-    meetsHousedReq,
-    // TODO: Add medicalResult(input).eligible once we can screen for Medi-Cal.
-    input.existingMedicalMe);
-
   const program = new Program();
-  // TODO: Replace this single example condition with a set of simplified
-  // conditions describing the separate eligibility requirements.
-  program.addCondition(new EligCondition('Example', eligible));
+  program.addCondition(
+    new EligCondition(`Disabled, blind or age ${cnst.ssiCapi.MIN_ELDERLY_AGE} or older`, meetsDisabilityReq));
```
Comment (trevorshannon) [inline src/site/_includes/js/eligibility.js:1787]: copy/paste error:

`cnst.ihss.MIN_ELDERLY_AGE`

**Review by trevorshannon — CHANGES_REQUESTED** [review]
Looks great!  Just one minor copy/paste issue.  Should be good to go after that is fixed.

**Review by trevorshannon — APPROVED** [review]
Thank you!

**Review by rajivrajiv — COMMENTED** [review]
ok

**Comment by rajivrajiv** [comment]
ihssResult() updated

**Pull request merged**